CONFLICTS.md

```diff
@@ -52,6 +52,13 @@ twig-bridge, var-dumper, var-exporter 6.0, which is not compatible with the curr
 
    Probably introduced in: https://github.com/symfony/symfony/pull/40811
 
+ - `symfony/dependency-injection:4.4.38|5.4.5`:
+   
+   These versions are causing a problem with mink session:
+  `InvalidArgumentException: Specify session name to get in vendor/friends-of-behat/mink/src/Mink.php:198`,
+   Psalm error: 
```

Comment:
This one really should be dealt with in the Sylius Psalm config, the test environment bug is a big thing to conflict against but this one really needs an application-level fix.  See https://github.com/symfony/symfony/issues/45627 and https://github.com/vimeo/psalm/issues/7703 for more context.